sandbox/libs/authn/src/main/java/org/opensearch/authn/DefaultObjectMapper.java

```diff
@@ -18,14 +20,18 @@
  * @opensearch.experimental
  */
 public class DefaultObjectMapper {
-    public static final ObjectMapper objectMapper = new ObjectMapper();
+    public static ObjectMapper objectMapper;
     public final static ObjectMapper YAML_MAPPER = new ObjectMapper(new YAMLFactory());
     private static final ObjectMapper defaulOmittingObjectMapper = new ObjectMapper();
 
     static {
+        objectMapper = JsonMapper.builder()
+            .enable(JsonParser.Feature.STRICT_DUPLICATE_DETECTION)
+            .disable(MapperFeature.CAN_OVERRIDE_ACCESS_MODIFIERS) // to prevent access denied exception by Jackson
+            .build();
+
         objectMapper.setSerializationInclusion(Include.NON_NULL);
         // objectMapper.enable(DeserializationFeature.FAIL_ON_TRAILING_TOKENS);
-        objectMapper.enable(JsonParser.Feature.STRICT_DUPLICATE_DETECTION);
```

Comment:
enable/ disable are deprecated and JsonMapper.builder() is suggested to be used for building a mapper